docs/experimental/framebuffer_example.py

```diff
@@ -0,0 +1,195 @@
+import numpy as np
+from fury import window, actor, lib, shaders
+import vtk
+
+
+def shader_custom_uniforms(actor : actor.Actor, shader_type : str):
+    """Eases the passing of uniform values to the shaders by returning ``actor.GetShaderProperty().GetVertexCustomUniforms()``, 
+    that give access to the ``SetUniform`` methods.
+    Parameters
+    ----------
+    actor : actor.Actor
+          Actor which the uniform values will be passed to.
+    shader_type : str
+          Shader type of the uniform values to be passed. It can be: 
+          * "vertex"
+          * "fragment"
+          * "geometry"
+          """
+    if shader_type == "vertex":
+        return actor.GetShaderProperty().GetVertexCustomUniforms()
+    elif shader_type == "fragment":
+        return actor.GetShaderProperty().GetFragmentCustomUniforms()
+    elif shader_type == "geometry":
+        return actor.GetShaderProperty().GetGeometryCustomUniforms()
+    else: raise ValueError("Shader type unknown.")
+
+
+
+# Don't mind these shaders, I just set them up to focus first on the framebuffer, then after on the shaders themselves
+billboard_vert_decl =  """/* Billboard  vertex shader declaration */
+in vec3 center;
+in vec2 in_tex;
+
+out vec3 centerVertexMCVSOutput;
+out vec3 normalizedVertexMCVSOutput;
+varying vec2 out_tex;"""
+
+billboard_vert_impl =  """/* Billboard  vertex shader implementation */
+centerVertexMCVSOutput = center;
+normalizedVertexMCVSOutput = vertexMC.xyz - center; // 1st Norm. [-scale, scale]
+float scalingFactor = 1. / abs(normalizedVertexMCVSOutput.x);
+float size = abs(normalizedVertexMCVSOutput.x) * 2;
+normalizedVertexMCVSOutput *= scalingFactor; // 2nd Norm. [-1, 1]
+vec2 billboardSize = vec2(size, size); // Fixes the scaling issue
+vec3 cameraRightMC = vec3(MCVCMatrix[0][0], MCVCMatrix[1][0], MCVCMatrix[2][0]);
+vec3 cameraUpMC = vec3(MCVCMatrix[0][1], MCVCMatrix[1][1], MCVCMatrix[2][1]);
+vec3 vertexPositionMC = center +
+    cameraRightMC * billboardSize.x * normalizedVertexMCVSOutput.x +
+    cameraUpMC * billboardSize.y * normalizedVertexMCVSOutput.y;
+out_tex = in_tex;
+gl_Position = MCDCMatrix * vec4(vertexPositionMC, 1.);"""
+
+
+frag_decl = """in vec3 normalizedVertexMCVSOutput;
+"""
+
+kde_func = """float kde(vec3 point, vec3 coord, float sigma){ 
+    return exp(-pow(length(abs(point - coord)), 2.0)/(2.0*sigma*sigma)); 
+}; """
+
+frag_impl = """
+vec3 k = vec3(kde(point0, vec3(gl_FragCoord.xy/res0, 0.0), 10.0)); 
+fragOutput0 = vec4(k, 0.5);"""
+
+
+frag = """
+
+varying vec2 out_tex;
+void main(){ 
+    gl_FragColor = vec4(1.0, 0.0, 1.0, 1.0); 
+}"""
+
+
+frag_2 = """
+
+varying vec2 out_tex;
+                    
+uniform sampler2D screenTexture;
+
+void main(){ 
+    vec4 texture = texture(screenTexture, out_tex);
```

Comment:
texture should be 2D vec right? What the meaning of out_text here?
try this

```
void main(){
    //vec2 texture = texture(screenTexture, out_tex);
   // gl_FragColor = texture;
  vec2 coords = gl_TexCoord[0].xy;
  vec3 normalColor = texture2D(screenTexture, coords).rgb;
  gl_FragColor = vec4(normalColor.r, normalColor.g, normalColor.b, 1);
    //gl_FragColor = vec4(0.0, 0.0, 1.0, 1.0);
}
```

Reply:
I was passing out_tex as texture coordinates, but I will try what you recommended